File: javascript/scrabble-score/scrabble-score.js
let scores = {
    "AEIOULNRST": 1,
    "DG": 2,
    "BCMP": 3,
    "FHVWY": 4,
    "K": 5,
    "JX": 8,
    "QZ": 10
};

const scoreLetter = (letter) => 
    Object.entries(scores)
        .find(([key, _]) => key.includes(letter))[1];

export const score = (word) => 
    [...word.toUpperCase()].reduce((acc, letter) => scoreLetter(letter) + acc, 0);

File: python/scrabble-score/scrabble_score.py
def score(word):

    return sum([score_letter(letter) for letter in word.upper()])

def score_letter(letter):

    letters_score = {
        'AEIOULNRST' : 1,
        'DG' : 2,
        'BCMP' : 3,
        'FHVWY' : 4,
        'K' : 5,
        'JX' : 8,
        'QZ' : 10
    }

    for (letters, value) in letters_score.items():
        if letter in letters:
            return value
